reusables/numbers.py

```diff
@@ -69,3 +82,106 @@ def roman_to_int(roman_string):
         last_number = _roman_dict[letter]
         value += _roman_dict[letter]
     return value
+
+
+def int_to_words(number, european=False):
+    """
+    Converts an integer or float to words.
+
+    :param The string, integer, or float to convert to words. The decimal
+        can only be up to three places long, and max number allowed is 999
+        decillion.
+    :param european: If the string uses the european style formatting, i.e.
+        decimal points instead of commas and commas instead of decimal points,
+        set this paramater to True
+    :return: The translated string
+    """
+    def ones(n):
+        if n != 0:
+            return _numbers[n]
+        else:
+            return ""
+
+    def tens(n):
+        teen = int("{0}{1}".format(n[0], n[1]))
+
+        if n[0] == 0:
+            return ones(n[1])
+        for k, v in _numbers.items():
+            if teen == k:
+                return v
+        else:
+            ten = _numbers[int("{0}0".format(n[0]))]
+            one = _numbers[n[1]]
+            return "{0}-{1}".format(ten, one)
+
+    def hundreds(n):
+        if n[0] == 0:
+            return tens(n[1:])
+        else:
+            t = tens(n[1:])
+            if t:
+                return "{0} hundred {1}".format(_numbers[n[0]], tens(n[1:]))
+            else:
+                return "{0} hundred".format(_numbers[n[0]])
+
+    decimal = ''
+    number = str(number)
+    
+    if european is False:
+        if "." in number:
+            decimal = number.split(".")[1]
+        number = number.split(".")[0].replace(",", "")
+
+    else:
+        if "," in number:
+            decimal = number.split(",")[1]
+        number = number.split(",")[0].replace(".", "")
+
+    if not number.isdigit():
+        raise ValueError("Number is not numeric")
+
+    if decimal and not decimal.isdigit():
+        raise ValueError("Number is not numeric")
```

Comment:
I would give this a separate error message, like 'decimal cannot be computed, malformed number'